modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -45,10 +45,20 @@
 		display: block;
 	}
 
+	.developer-name {
```

Comment:
Provide more context for this selector. If the style is specific to an element, I need to make sure that the style is only applied to that element.


```suggestion
	.solution-search-results-card .card-image-title-container .developer-name {
```